packages/KTable/KTable.vue

```diff
@@ -173,22 +195,39 @@ export default {
     },
 
     trlisteners () {
-      return pluckListeners('row:', this.$listeners)
+      return (entity, type) => {
+        const pluckedListeners = pluckListeners('row:', this.$listeners)(entity, type)
+
+        return {
+          ...pluckedListeners,
+          mouseup (e) {
+            if (e.target.tagName === 'TD' && pluckedListeners['mouseup']) {
+              pluckedListeners['mouseup'](e, entity, type)
+            }
+          }
+        }
+      }
     }
   }
 }
 </script>
 
 <style scoped lang="scss">
 @import '~@kongponents/styles/_variables.scss';
-
-table.k-table {
-  border-collapse: collapse;
-}
-
 .k-table {
   width: 100%;
   max-width: 100%;
+  border-collapse: collapse;
+
+  &.side-border {
+    border-collapse: separate;
+    border-spacing: 0 4px;
+
+    tbody tr {
+      border: none;
+      box-shadow: -2px 0 0 var(--KTableBorder, var(--steal-300, color(steal-300)));
```

Comment:
Actually looks like the default changes on hover. So maybe if `hasHover` then default should be `steal-200` and `steal-300` on hover
https://share.goabstract.com/edbb70e6-5126-4195-b523-80ab300e1459?mode=build&sha=5447ee919f352c8b9e0029181760b55ac98b844e